src/03-modality-agnostic-files.md

```diff
@@ -383,6 +385,34 @@ meg/sub-control01_task-rest_split-01_meg.nii.gz	1877-06-15T12:15:27
 meg/sub-control01_task-rest_split-02_meg.nii.gz	1877-06-15T12:15:27
 ```
 
+## Sessions file
+
+Template:
+
+```Text
+sub-<label>/
+    ses-<label>/
+        sub-<label>_sessions.tsv
```

Comment:
```suggestion
sub-<label>/
    sub-<label>_sessions.tsv
```

Reply:
@remi-gau This still needs fixing. sessions.tsv is not inside a session directory.

Reply:
Thanks. I scrolled too quickly this morning.